### PR TITLE
Added tabulate+<

### PR DESCRIPTION
**Pull request by guilhermehas**
`tabulate⁺<` is more general than `tabulate⁺`

### CODE REVIEW

File: src/Data/List/Relation/Unary/AllPairs/Properties.agda [hunk 1]
```diff
@@ -115,12 +115,16 @@ module _ {R : Rel A ℓ} where
 
 module _ {R : Rel A ℓ} where
 
+  tabulate⁺< : ∀ {n} {f : Fin n → A} → (∀ {i j} → i F.< j → R (f i) (f j)) →
```
Comment (MatthewDaggitt) [inline src/Data/List/Relation/Unary/AllPairs/Properties.agda:118]: Could we call this `tabulate⁺-<`?

File: src/Data/List/Relation/Unary/AllPairs/Properties.agda [hunk 2]
```diff
@@ -115,12 +115,16 @@ module _ {R : Rel A ℓ} where
...
+              AllPairs R (tabulate f)
+  tabulate⁺< {zero}  fᵢ~fⱼ = AllPairs.[]
```
Comment (MatthewDaggitt) [inline src/Data/List/Relation/Unary/AllPairs/Properties.agda:120]: I don't think we need the `AllPairs.` qualifiers anywhere in this proof?

**Review by MatthewDaggitt — CHANGES_REQUESTED** [review]
Otherwise looks good!

File: src/Data/List/Relation/Unary/AllPairs/Properties.agda (hunk 1 above)
Comment (guilhermehas) [reply src/Data/List/Relation/Unary/AllPairs/Properties.agda]: I renamed it.

File: src/Data/List/Relation/Unary/AllPairs/Properties.agda (hunk 2 above)
Comment (guilhermehas) [reply src/Data/List/Relation/Unary/AllPairs/Properties.agda]: I removed it.

**Review by MatthewDaggitt — APPROVED** [review]

**Pull request merged**